# TO DO
## mat2mat
* pre-processingL
	* regions -> tab
	* non-csm/mat -> mtx
* isStrand
* min fraction of region overlap
* mean counts when small regions in the middle and many overlap?
* selected contigs
* unconventional/unmatsched contig names
* cell -> sample
* improve mtx reading

## Other functions to add
* get statistics, qc, downstream analysis
	* regions: 
		* sizes (mean, median, max, min, hist)
		* names? (e.g. gene names; TSS; genomic features --> grouping into statistics)
	* mat: clustering? other statistics?
	* cells: "QC" (regions per cell, cells per region, etc.)
* forms of mat
	- [x] mtx 
	- [] csv/tsv?
	- [] in python env, arrays?
* DOCUMENTATION
* NAMING CHANGES
